### PR TITLE
Map Edits (snowdin, caves, z2, academy, ministation)

### PR DESCRIPTION
**Pull request by MMMiracles**
Snowdin now actually opens (double // on the turf paths honk honk).

Removed the snow plating entirely, replaced with snow subtype of asteroid, maps (affected maps edited).

Uses new snow sprites credited to Nienhaus (needed to so it'd fall in line with the standards that the basalt and asteroid turfs had).

Cave's basalt rocks should now actually break to the volcanic flooring instead of asteroid.

### CODE REVIEW

**Comment by MMMiracles** [comment]
MINISTATIOOOON

**Comment by swankcookie** [comment]
Thanks for this.  I thought I'd broken my master branch...

File: _maps/map_files/MiniStation/MiniStation.dmm [hunk 1]
```diff
@@ -1,8 +1,8 @@
 "aa" = (/turf/space,/area/space)
-"ab" = (/obj/docking_port/stationary{dheight = 9; dir = 2; dwidth = 5; height = 22; id = "syndicate_n"; name = "north of station"; turf_type = /turf/simulated/floor/plating/snow; width = 18},/turf/space,/area/space)
+"ab" = (/obj/docking_port/stationary{dheight = 9; dir = 2; dwidth = 5; height = 22; id = "syndicate_n"; name = "north of station"; turf_type = turf/simulated/floor/plating/asteroid/snow; width = 18},/turf/space,/area/space)
```
Comment (tkdrg) [inline _maps/map_files/MiniStation/MiniStation.dmm]: Missing the `/` in `/turf` for all of the ministation `turf_type`s.

File: _maps/map_files/MiniStation/MiniStation.dmm [hunk 2]
```diff
@@ -1,8 +1,8 @@
 "aa" = (/turf/space,/area/space)
-"ab" = (/obj/docking_port/stationary{dheight = 9; dir = 2; dwidth = 5; height = 22; id = "syndicate_n"; name = "north of station"; turf_type = /turf/simulated/floor/plating/snow; width = 18},/turf/space,/area/space)
+"ab" = (/obj/docking_port/stationary{dheight = 9; dir = 2; dwidth = 5; height = 22; id = "syndicate_n"; name = "north of station"; turf_type = turf/simulated/floor/plating/asteroid/snow; width = 18},/turf/space,/area/space)
 "ac" = (/obj/effect/landmark{name = "carpspawn"},/turf/space,/area/space)
 "ad" = (/turf/simulated/mineral/random/low_chance,/area/mine/explored)
-"ae" = (/obj/docking_port/stationary{dheight = 9; dir = 2; dwidth = 5; height = 22; id = "syndicate_ne"; name = "northeast of station"; turf_type = /turf/simulated/floor/plating/snow; width = 18},/turf/space,/area/space)
+"ae" = (/obj/docking_port/stationary{dheight = 9; dir = 2; dwidth = 5; height = 22; id = "syndicate_ne"; name = "northeast of station"; turf_type = turf/simulated/floor/plating/asteroid/snow; width = 18},/turf/space,/area/space)
```
Comment (tkdrg) [inline _maps/map_files/MiniStation/MiniStation.dmm]: Same as above

File: _maps/map_files/MiniStation/MiniStation.dmm [hunk 3]
```diff
@@ -1677,7 +1677,7 @@
 "Gm" = (/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{dir = 4},/turf/simulated/floor/plasteel{dir = 1; icon_state = "warning"},/area/engine/gravity_generator)
 "Gn" = (/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"},/obj/machinery/atmospherics/pipe/manifold/scrubbers/hidden{dir = 4},/turf/simulated/floor/plasteel{dir = 1; icon_state = "warning"},/area/engine/gravity_generator)
 "Go" = (/turf/simulated/floor/plasteel{dir = 5; icon_state = "warning"},/area/engine/gravity_generator)
-"Gp" = (/obj/docking_port/stationary{dheight = 9; dir = 2; dwidth = 5; height = 22; id = "syndicate_sw"; name = "southwest of station"; turf_type = /turf/simulated/floor/plating/snow; width = 18},/turf/space,/area/space)
+"Gp" = (/obj/docking_port/stationary{dheight = 9; dir = 2; dwidth = 5; height = 22; id = "syndicate_sw"; name = "southwest of station"; turf_type = turf/simulated/floor/plating/asteroid/snow; width = 18},/turf/space,/area/space)
```
Comment (tkdrg) [inline _maps/map_files/MiniStation/MiniStation.dmm]: Same again

File: _maps/map_files/MiniStation/MiniStation.dmm [hunk 4]
```diff
@@ -1690,7 +1690,7 @@
 "Gz" = (/obj/structure/cable{d1 = 2; d2 = 4; icon_state = "2-4"},/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"},/turf/simulated/floor/plasteel,/area/engine/gravity_generator)
 "GA" = (/obj/machinery/door/window{dir = 8; name = "Tool Storage"; req_access_txt = "150"},/turf/simulated/floor/plasteel/shuttle{icon_state = "shuttlefloor4"},/area/shuttle/syndicate)
 "GB" = (/turf/simulated/floor/plasteel,/area/engine/gravity_generator)
-"GC" = (/obj/docking_port/stationary{dheight = 9; dir = 2; dwidth = 5; height = 22; id = "syndicate_se"; name = "southeast of station"; turf_type = /turf/simulated/floor/plating/snow; width = 18},/turf/space,/area/space)
+"GC" = (/obj/docking_port/stationary{dheight = 9; dir = 2; dwidth = 5; height = 22; id = "syndicate_se"; name = "southeast of station"; turf_type = turf/simulated/floor/plating/asteroid/snow; width = 18},/turf/space,/area/space)
```
Comment (tkdrg) [inline _maps/map_files/MiniStation/MiniStation.dmm]: Same

File: _maps/map_files/MiniStation/MiniStation.dmm [hunk 5]
```diff
@@ -1705,7 +1705,7 @@
 "GO" = (/obj/structure/table,/obj/item/weapon/gun/syringe{pixel_x = 1; pixel_y = 2},/turf/simulated/floor/plasteel/shuttle{icon_state = "shuttlefloor4"},/area/shuttle/syndicate)
 "GP" = (/obj/machinery/telecomms/processor/preset_one,/turf/simulated/floor/bluegrid{icon_state = "dark"; name = "Mainframe Floor"; nitrogen = 100; oxygen = 0; temperature = 80},/area/tcommsat/computer)
 "GQ" = (/obj/machinery/telecomms/broadcaster/preset_left/ministation,/turf/simulated/floor/bluegrid{icon_state = "dark"; name = "Mainframe Floor"; nitrogen = 100; oxygen = 0; temperature = 80},/area/tcommsat/computer)
-"GR" = (/obj/docking_port/stationary{dheight = 9; dir = 2; dwidth = 5; height = 22; id = "syndicate_s"; name = "south of station"; turf_type = /turf/simulated/floor/plating/snow; width = 18},/turf/space,/area/space)
+"GR" = (/obj/docking_port/stationary{dheight = 9; dir = 2; dwidth = 5; height = 22; id = "syndicate_s"; name = "south of station"; turf_type = turf/simulated/floor/plating/asteroid/snow; width = 18},/turf/space,/area/space)
```
Comment (tkdrg) [inline _maps/map_files/MiniStation/MiniStation.dmm]: Yep same thing

File: code/game/turfs/simulated/floor/misc_floor.dm [hunk 6]
```diff
@@ -43,46 +43,38 @@
...
-	icon_state = "gravsnow_surround"
-
 /turf/simulated/floor/plating/ice
 	name = "ice sheet"
 	desc = "A sheet of solid ice. Looks slippery."
 	icon = 'icons/turf/snow.dmi'
 	icon_state = "ice"
-	baseturf = /turf/simulated/floor/plating/snow
+	temperature = 180
+	baseturf = /turf/simulated/floor/plating/ice
```
Comment (tkdrg) [inline code/game/turfs/simulated/floor/misc_floor.dm]: its baseturf is itself?

Comment (MMMiracles) [inline code/game/turfs/simulated/floor/misc_floor.dm]: either that or it defaults to space, which probably doesn't make much sense.

**Pull request merged**